gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -38,7 +38,7 @@ module MAPL_HistoryGridCompMod
   use mapl_RegridMethods
   use MAPL_newCFIOitemVectorMod
   use MAPL_newCFIOitemMod
-  use pFIO_ClientManagerMod, only: o_Clients
+  use pFIO
```

Comment:
Just a question: do we need to use all of pFIO here? Or just one routine a la:
```fortran
use pFIO, only: shavebits
````

I'm not sure what @tclune thinks of this. I don't see anything in the [Style Guide](https://github.com/GEOS-ESM/MAPL/wiki/MAPL-Coding-standards) for how to handle use/use-only stuff...

Reply:
Oh I'm an idiot. it's in the file. Time to end the day! :)

Reply:
I am making changes to use smallest set of routines from pfio